.ci/deps.sh

```diff
@@ -45,6 +45,13 @@ go get -u golang.org/x/tools/cmd/gotype
 # Ruby commands
 bundle install
 
+pip install appdirs
```

Comment:
You need to do it after the for loop.
As the for loop runs `pyenv local $dep_version 2.7.10` which sets the python version to use for this circleci container